Mathlib/Analysis/NormedSpace/Star/Matrix.lean

```diff
@@ -70,3 +86,79 @@ theorem entrywise_sup_norm_bound_of_unitary {U : Matrix n n 𝕜} (hU : U ∈ Ma
 #align entrywise_sup_norm_bound_of_unitary entrywise_sup_norm_bound_of_unitary
 
 end EntrywiseSupNorm
+
+noncomputable section EuclideanSpaceOpNorm
+
+variable [Fintype n] [DecidableEq n] [IsROrC 𝕜]
+
+/-- The natural star algebra equivalence between matrices and continuous linear endomoporphisms
+of Euclidean space induced by the orthonormal basis `EuclideanSpace.basisFun`. -/
+def Matrix.toEuclideanClm :
```

Comment:
Please add some `simp` lemmas along the lines of

https://github.com/leanprover-community/mathlib4/blob/b180173c90858e2031655e09a7165dbf1048324a/Mathlib/Analysis/InnerProductSpace/PiL2.lean#L1018C1-L1029